packages/mui-material/src/Tabs/ScrollbarSize.js

```diff
@@ -21,21 +21,22 @@ export default function ScrollbarSize(props) {
   const scrollbarHeight = React.useRef();
   const nodeRef = React.useRef(null);
 
-  const setMeasurements = () => {
-    scrollbarHeight.current = nodeRef.current.offsetHeight - nodeRef.current.clientHeight;
+  const setMeasurements = (element) => {
+    scrollbarHeight.current = element.offsetHeight - element.clientHeight;
   };
 
   React.useEffect(() => {
```

Comment:
Would it be a simpler fix? 

```suggestion
  useEnhancedEffect(() => {
```

Reply:
@oliviertassinari Tried to avoid synchronous effect if it can be avoided in another way because it can impose unnecessary impact on performance. mount and unmount would not be called frequently so it would be ok to do that in this feature I think. Thanks for the review.

Reply:
I feel that the synchronous effect makes more sense since we want to detect layout changes.